pages/index.tsx

```diff
@@ -10,6 +14,137 @@ const Home = () => {
       </Head>
       <Hero />
       <WhyKedro />
+      <section className={style.features}>
+        <h3 className={style.sectionTitle}>Features</h3>
+        <FeatureDetails
+          buttonLink="https://demo.kedro.org/"
+          buttonText="Explore Live Demo"
+          iframeLink="https://demo.kedro.org/"
+          subtitle={
+            <>
+              <a
+                href="https://github.com/kedro-org/kedro-viz"
+                rel="noopener noreferrer"
+                target="_blank"
+              >
+                Kedro&#39;s pipeline visualisation plugin
+              </a>{' '}
+              shows a blueprint of your developing data and machine-learning
+              workflows, provides data lineage, keeps track of machine-learning
+              experiments and makes it easier to collaborate with business
+              stakeholders.
+            </>
+          }
+          title="Pipeline visualisation"
+        />
+        <FeatureDetails
+          altText="Placeholder"
+          assetPosition="right"
+          buttonLink="https://kedro.readthedocs.io/en/stable/02_get_started/05_example_project.html#project-directory-structure"
+          buttonText="Learn more"
+          imageSrc={placeholder}
+          subtitle={
+            <>
+              You can standardise how configuration, source code, tests,
+              documentation, and notebooks are organised with an adaptable,
+              easy-to-use project template.{' '}
+              <a
+                href="https://kedro.readthedocs.io/en/stable/07_extend_kedro/05_create_kedro_starters.html"
+                rel="noopener noreferrer"
+                target="_blank"
+              >
+                Create your cookie cutter project templates with Starters.
+              </a>
+            </>
+          }
+          title="Project Template"
+        />
+        <FeatureDetails
+          altText="Placeholder"
+          assetPosition="left"
+          imageSrc={placeholder}
```

Comment:
Similar to above, this image is also avaliable on Figma and can be added in directly
<img width="1156" alt="image" src="https://user-images.githubusercontent.com/27775658/155740355-c229f5da-941b-4db9-9d21-030c0538fe91.png">

Reply:
Added.